documentation/decisions/adr/0004-db-planning.md

```diff
@@ -0,0 +1,63 @@
+# DB Planning
+
+- **Status:** Approved
+- **Last Modified:** 2023-07-05
+- **Related Issue:** [#27](https://github.com/HHS/grants-api/issues/104) 
+- **Deciders:** Lucas Brown, Aaron Couch, Gina Carson, Andy Cochran
+
+## Context and Problem Statement
+
+This ADR is to formalize a series of architectural decisions about how data is stored, including the type of database we will use and the platform we'll use host it. This ADR will contain the evaluation and selection of the type of the database, the database management system (DBMS), and database hosting service. 
+
+## Decision Drivers <!-- RECOMMENDED -->
+Type of Database:
+
+- Nature of the NOFO and grant data that we're storing
+- Volume of data and performance considerations
+
+
+DBMS Selection:
+
+- Hosting options
+- Expected access patterns (i.e. heavy read or write transactions)
+- Ease of creating mock data
+- Trusted libraries that support interfacing with this database in the language we've chosen for our API
```

Comment:
```suggestion
- Trusted libraries that support interfacing with this database in the language we've chosen for our API
- An open source version of this DBMS is available for self hosting or running locally
```

You touched on this in the pros for Postgres, but thought it would be good to promote to a decision driver as well